owncloudApp/src/main/res/values/strings.xml

```diff
@@ -595,8 +596,6 @@
     <string name="welcome_feature_4_text">Your pictures/videos automatically uploaded</string>
     <string name="welcome_feature_5_title">Video streaming</string>
     <string name="welcome_feature_5_text">Play your videos without downloading them</string>
-    <string name="welcome_feature_11_title">End of support for servers lower than 10</string>
```

Comment:
We need to remove these entries from all the translations, otherwise we will get a lint error, please check the logs in https://app.bitrise.io/build/fe1eeaea26729c24#?tab=log , you will see something like

![Screenshot 2020-01-13 at 14 33 39](https://user-images.githubusercontent.com/15261363/72259947-ef93b480-3611-11ea-9c9b-7923fbaf97d1.png)